src/applications/vaos/components/express-care/FormButtons.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ProgressButton from 'platform/forms-system/src/js/components/ProgressButton';
+import LoadingButton from 'platform/site-wide/loading-button/LoadingButton';
+
+export default function FormButtons({
```

Comment:
Is this any different from the ones used on the regular form? Could we share between the two forms?

Reply:
#### Express care buttons
<img width="509" alt="Screen Shot 2020-07-27 at 9 49 04 AM" src="https://user-images.githubusercontent.com/786704/88569272-0c606a00-cfef-11ea-91e1-603d71b5ac49.png">

#### New appointment buttons
![image](https://user-images.githubusercontent.com/786704/88569224-fc488a80-cfee-11ea-8d21-18f3c570f549.png)

I was going back and forth on this, but there are a couple design differences:
- On express care, the button width is only as much as the text + some padding, whereas they are equal and take up half of the grid on the regular flow
- On the regular flow the text is consistently "Back" and "Continue", whereas it is different on each page for express care
- On express care the buttons are missing the double chevron icons (`<<` and `>>`)

Maybe @rtwell can chime in on if we should match the design of express care buttons to the existing flow, or if it's okay that they are slightly different.

Reply:
Ah, I see. For the first point, I think all of Ryan's designs have had the buttons only be as wide as the text, we just didn't do that for the main form. So I would imagine we would want that to be consistent in both places. But yeah, not sure about the other two.

Reply:
Updated to use a shared `<FormButtons>` component with non-full-width buttons and customizable text.  The chevrons can be updated later, but left them in for new appointment flow.